src/app/wizard/step/cluster/component.ts

```diff
@@ -258,7 +260,7 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
   }
 
   updateCNIPluginOptions() {
-    if (this.controlValue(Controls.CNIPlugin) === 'cilium') {
+    if (this.controlValue(Controls.CNIPlugin) === CNIPlugin.cilium) {
```

Comment:
Please also change the enum keys to use upper case. The internal value can be the same as now.